packages/main/src/plugin/kubernetes/contexts-manager.ts

```diff
@@ -364,7 +366,7 @@ export class ContextsManager {
     };
   }
 
-  private createPodInformer(kc: KubeConfig, namespace: string, context: KubeContext): Informer<V1Pod> {
+  private createPodInformer(kc: KubeConfig, namespace: string, context: KubeContext): CancellableInformer {
```

Comment:
Why are we losing the generic type, is it important ?

Reply:
I don't think so:

The `V1Pod` is necessary when calling `this.createInformer<V1Pod>`, to have he `onAdd` and other options have the matching type. But the result of `createPodInformer` is stored in an `Informer<KubernetesObject>`; the specific type is never used on this side.

Reply:
Okey thanks